apps/site/pages/components/molecules/gift.mdx

```diff
@@ -0,0 +1,252 @@
+---
+description: It displays an additional item to purchase as a gift.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/ProductCardComponent___e6502d57cf7e5bf247f86b64577d4141.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+
+import { Badge, Gift, GiftContent, GiftImage, Icon, Price } from '@faststore/ui'
+import { TagIcon } from '@faststore/components'
+
+import { product } from '../../../mocks/product'
+import { useMemo } from 'react'
+
+<header>
+
+# Gift
+
+It displays an additional item to purchase as a gift.
+
+</header>
+
+The Gift component is a compound of the following:
+
+- `Gift` - renders a `<div>` as wrapper with an `Icon`, `GiftImage`, and `GiftContent`.
+- `GiftImage` - renders a `<div>` as wrapper with an `Image`.
+- `GiftContent` - renders a `<div>` as wrapper with the gift information as children.
+
+## Overview
```

Comment:
You could add different variations for the `Gift` here :)
Examples:
- Change the icon
- Without icon
- Without badge

Reply:
Nice! done [here](https://github.com/vtex/faststore/pull/1610/commits/5b978074928997b49e293a04411bedc7f011c5b6).